pkgs/development/python-modules/dragonmapper/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  unittestCheckHook,
+  hanzidentifier,
+  zhon,
+}:
+
+buildPythonPackage rec {
+  pname = "dragonmapper";
+  version = "0.2.7";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tsroten";
+    repo = "dragonmapper";
+    rev = "v${version}";
+    hash = "sha256-/02vcjcsUpQA1R1hcp34g/MSzNrKwuEyY5ERQQ5Vemw=";
+  };
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    hanzidentifier
+    zhon
+  ];
+
+  nativeCheckInputs = [ unittestCheckHook ];
+
+  pythonImportsCheck = [ "dragonmapper" ];
+
```

Comment:
```suggestion
passthru.updateScript = nix-update-script { };
```

Reply:
this is not necessary as python modules all get default updateScript

Reply:
good to know. Thanks!